doc/usage/extensions/autosectionlabel.rst

```diff
@@ -39,6 +39,13 @@ Configuration
    avoiding ambiguity when the same section heading appears in different
    documents.
 
+   When ``index.rst`` doesn't reside at your docs root directory, the prefix
+   has to contain any directories relative to it.
```

Comment:
"document name" is already defined in glossary. It's common word for sphinx users, I believe. So what we should do is add `:term:` role to above description.
https://www.sphinx-doc.org/en/master/glossary.html#term-document-name